code/drasil-printers/Language/Drasil/TeX/Helpers.hs

```diff
@@ -46,14 +46,17 @@ commandD s c = pure (H.bslash TP.<> text s) <> br c
 command1o :: String -> Maybe String -> String -> D
 command1o s o c = pure $ (H.bslash TP.<> text s) TP.<> maybe TP.empty H.sq o TP.<> H.br c
 
--- no braces!
 command1oD :: String -> Maybe D -> D -> D
-command1oD s o c = pure (H.bslash TP.<> text s) <> maybe empty sq o <> c
+command1oD s o c = pure (H.bslash TP.<> text s) <> maybe empty sq o <> br c
```

Comment:
So this function was explicitly documented to produce no braces -- you removed the comment and the braces. Why? [I'm not saying it is necessarily wrong, just that it needs justification]

Reply:
It was only used in `item'`, and its counterpart `item` (the same without an extra argument) used the braces, so it made sense to make this outlier consistent with the rest of the family of `command` functions.